scripts/istat/geos/README.MD

```diff
@@ -0,0 +1,23 @@
+# Importing the ItalySTAT: List of codes and names of territorial units into Data Commons
+
+This directory contains artifacts required for importing the the ISTAT data into Data Commons, along with scripts used to generate these artifacts
+
+## Artifacts:
+* [Elenco-codici-statistici-e-denominazioni-al-01_01_2020_it.csv][1]: original CSV file
```

Comment:
Can consider having a raw/ and cleaned/ separation between original and cleaned csvs

Reply:
done